.devcontainer/devcontainer.json

```
@@ -58,22 +40,13 @@
+                "ms-azuretools.vscode-docker"
             ]
         }
     },
-    // Explicitly forward sphinx-autobuild port (auto-detection disabled above)
-    "forwardPorts": [
-        8000
-    ],
-    // Create host-side dirs needed for bind mounts before the container starts
-    "initializeCommand": "mkdir -p ${localEnv:HOME}/.config/terminal-config ${localEnv:HOME}/.claude",
+    // Create the config folder for the bash-config feature and uv cache
+    "initializeCommand": "mkdir -p ${localEnv:HOME}/.config/terminal-config",
```

Biome (2.4.16)

[error] The key initializeCommand was already declared.

(lint/suspicious/noDuplicateObjectKeys)